FirebaseAuthUI/README.md

```diff
@@ -69,8 +70,9 @@ authUI?.delegate = self
 @import FirebaseAuthUI
 ...
 [FIRApp configure];
-FIRAuthUI *authUI = [FIRAuthUI defaultAuthUI];
-authUI.delegate = self; // Set the delegate to receive callback.
+FIRAuthUI *authUI = [FIRAuthUI default];
```

Comment:
This should still be `[FIRAuthUI defaultAuthUI]` in objc; we didn't rename the method, Swift's objc import rules changed in Swift 3.